src/operation.js

```diff
@@ -114,20 +111,17 @@ export class Operation {
    * was used to create the operation (i.e. the `opts` parameter to most ops).
    *
    * @param {xdr.Operation}   operation - An XDR Operation.
-   * @param {boolean}         [withMuxing] - Indicates that if the operation
-   *     contains M... addresses, they should be interpreted fully as muxed
-   *     accounts. By default, this option is disabled until muxed accounts are
-   *     mature.
+   * @param {bool}    [=true] - Indicates any properties in this
```

Comment:
Why do we keep this boolean property even though it is being removed?

Reply:
Ctrl+F mistake :smile:

Reply:
Fixed in 2f7da085bf49f2b2ebe682b1217f56dc543c2c40.